.ci/bump-stack-version.sh

```diff
@@ -24,6 +24,25 @@ else
 	SED="sed -i"
 fi
 
+echo "Update stack with version ${VERSION} in .stack-version"
+echo "${VERSION}" > .stack-version
+git add .stack-version
+
+echo "Update stack with version ${VERSION} in Go files"
+find . -name 'defaults.go' -path './internal/common/*' -print0 |
+	while IFS= read -r -d '' FILE ; do
+		${SED} -E -e "s#(var BeatVersionBase = (\"))[0-9]+\.[0-9]+\.[0-9]+(-[a-f0-9]{8})?#\1${VERSION}#g" $FILE
+		git add $FILE
+	done
+
+echo "Update stack with version ${VERSION} in Jenkinsfile"
+find . -name 'Jenkinsfile' -path './.ci/*' -print0 |
```

Comment:
Ditto, there is only one `Jenkinsfile`, a simple sed should be enough?

Reply:
Done in https://github.com/elastic/e2e-testing/pull/1351/commits/fe83a86b26f9e5a055985cabcd29423b6a0ba292